modules/psap-special-resource-operator-metrics.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="special-resource-operator-metrics_{context}"]
+= Prometeus Special Resource Operator metrics
+
+
+The Special Resource Operator (SRO) exposes the following Prometheus metrics through the `metrics` service:
+
+|===
+|Metric Name |Description
+
+|`sro_used_nodes`
+|Returns the nodes that are running pods created by a SRO custom resource (CR). This metric is available for `DaemonSet` and `Deployment` objects only. 
+
+|`sro_kind_completed_info`
+|Represents whether a `kind` of SRO CR object has been successfully uploaded in the cluster (value `1`) or not (value `0`). Examples of objects are `DaemonSet`, `Deployment` or `BuildConfig`.
```

Comment:
maybe:
"whether a `kind` of an object defined by the Helm charts of SRO CR?
SRO CR itself is an object. It contains pointer to helm chart/s. Inside Helm charts there is a definition of one of more objects to be uploaded. "Kind" refers to those objects